lib/models/asyncapi.js

```diff
@@ -5,7 +5,8 @@ const Info = require('./info');
 const Server = require('./server');
 const Channel = require('./channel');
 const Components = require('./components');
-
+// eslint-disable-next-line no-unused-vars
+const Schema = require('./schema');
```

Comment:
This can be removed now

Reply:
@jonaslagoni made the changes. thanks! couldn't have done it without you.